app/src/main/res/layout/activity_transaction_detail.xml

```diff
@@ -0,0 +1,279 @@
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/transaction_detail_amount"
+        tools:text="+5.50" />
+
+    <com.tari.android.wallet.ui.component.CustomFontTextView
+        android:id="@+id/tx_fee_label"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/transaction_fee"
+        android:textColor="@color/light_gray"
+        android:textSize="14sp"
+        app:customFont="AVENIR_LT_STD_ROMAN"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/tx_detail_tx_fee" />
+
+    <RelativeLayout
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/white"
+        android:layout_marginTop="32dp"
+        android:elevation="5dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/tx_fee_label">
+
+        <LinearLayout
+            android:id="@+id/transaction_detail_body"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:paddingStart="24dp">
+
+            <com.tari.android.wallet.ui.component.CustomFontTextView
+                android:id="@+id/from_label"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="32dp"
+                android:text="@string/from"
+                android:textColor="@color/light_gray"
+                android:textSize="14sp"
+                app:customFont="AVENIR_LT_STD_MEDIUM" />
+
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="8dp">
+
+                <include
+                    android:id="@+id/transaction_detail_emoji_summary"
+                    layout="@layout/emoji_id_summary"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content" />
+
+                <com.tari.android.wallet.ui.component.CustomFontTextView
+                    android:id="@+id/tx_detail_add_contact"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentEnd="true"
+                    android:layout_marginEnd="24dp"
+                    android:text="@string/add_contact"
+                    android:textColor="@color/home_bg_gradient_center"
+                    app:customFont="AVENIR_LT_STD_MEDIUM" />
+            </RelativeLayout>
+
+
```

Comment:
Please don't use double empty lines if you're not emphasizing something - and in that case you can use XML comments.